packager/util/config.go

```diff
@@ -50,6 +54,16 @@ type URLPattern struct {
 	SamePath               *bool
 }
 
+type ACMEConfig struct {
+	Prod	*ACMEServerConfig
+	Staging	*ACMEServerConfig
```

Comment:
I'd rename to Development; matches our `-development` command-line flag. We have plans for a `-staging` flag (#188) but I think that'll use production config in every way except the cert-url it writes into the SXG.

Reply:
Development sounds reasonable.